src/EnergyPlus/PlantLoadProfile.hh

```diff
@@ -65,27 +65,38 @@ struct EnergyPlusData;
 
 namespace PlantLoadProfile {
 
+    enum class PlantLoopFluidType
+    {
+        Invalid = -1,
+        Water,
+        Steam,
+        Num
+    };
```

Comment:
Is glycol not an option? or is that just another form of water?

Reply:
I think it is another form of water. I set the enum `Water` as per the NodeFluidType of PlantLoadProfile's inlet/outlet node [here](https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/PlantLoadProfile.cc#L361).